api/src/main/resources/notebooks.yaml

```diff
@@ -47,98 +44,111 @@ securityDefinitions:
+        "409":
+          description: Runtime cannot be stopped
+          content:
+            application/json:
+              schema:
+                $ref: "#/definitions/ErrorReport"
+        "500":
           description: Internal Error
-          schema:
-            $ref: '#/definitions/ErrorReport'
-
-  '/api/cluster/{googleProject}/{clusterName}/start':
+          content:
+            application/json:
+              schema:
+                $ref: "#/definitions/ErrorReport"
+  /api/google/v1/runtimes/{googleProject}/{name}/start:
     post:
-      summary: Starts a Dataproc cluster
-      description: |
-        Starts the instances of a stopped Dataproc cluster.
-      operationId: startCluster
+      summary: Starts a Dataproc cluster or Google Compute Engine instance
+      description: Starts the a stopped runtime
+      operationId: startRuntime
       tags:
-        - cluster
+        - runtimes
       parameters:
         - in: path
           name: googleProject
           description: googleProject
           required: true
           type: string
         - in: path
-          name: clusterName
-          description: clusterName
+          name: name
+          description: runtimeName
           required: true
           type: string
       responses:
-        '202':
-          description: Cluster start request accepted
-        '403':
-          description: User does not have permission to perform action on cluster
-        '404':
-          description: Cluster not found
-          schema:
-            $ref: '#/definitions/ErrorReport'
-        '409':
-          description: Cluster cannot be started
-          schema:
-            $ref: '#/definitions/ErrorReport'
-        '500':
+        "202":
+          description: Runtime start request accepted
+        "403":
+          description: User does not have permission to perform action on runtime
+        "404":
+          description: Runtime not found
+          content:
+            application/json:
+              schema:
+                $ref: "#/definitions/ErrorReport"
+        "409":
+          description: Runtime cannot be started
+          content:
+            application/json:
+              schema:
+                $ref: "#/definitions/ErrorReport"
+        "500":
           description: Internal Error
-          schema:
-            $ref: '#/definitions/ErrorReport'
+          content:
+            application/json:
+              schema:
+                $ref: "#/definitions/ErrorReport"
 
-  '/notebooks/{googleProject}/{clusterName}/setCookie':
+  "/proxy/{googleProject}/{clusterName}/setCookie":
     get:
-      summary: (Deprecated) Sets a browser cookie needed to authorize connections to a Jupyter notebook
-      description: |
-        If using Google token-based authorization to a Jupyter notebook, the Leo proxy accepts a
+      summary: Sets a browser cookie needed to authorize connections to a Dataproc
```

Comment:
is this un-deprecated?

Reply:
The "proxy" endpoint is not deprecated, the /notebooks one was